ports/stm32/eth.c

```diff
@@ -78,7 +90,7 @@
 #define TX_DESCR_0_TER_Pos      (21)
 #define TX_DESCR_0_TCH_Pos      (20)
 #define TX_DESCR_1_TBS1_Pos     (0)
-
+#endif
```

Comment:
Would be good to keep these blank separation lines, ie:
```c
#define TX_DESCR_1_TBS1_Pos     (0)
#endif 

// Configuration values
```
There are also other instances of this below.

Reply:
Restored the separation lines